### PR TITLE
Fix/import contacts with duplicates

### PR DESCRIPTION
**Pull request by musamusa**
image

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -1404,3 +1404,7 @@ def traces_sampler(sampling_context):  # pragma: no cover
 
 CORS_ALLOW_ALL_ORIGINS = True
 CORS_ALLOW_METHODS = ["GET"]
+
+# Contacts import via excel
+# if set to True will not raise error on duplicate, instead will use last row
+IGNORE_DUPLICATES_CONTACT_IMPORT = False
```
Comment (teehamaral) [inline temba/settings_common.py:1410]: Can we have this value getting from an environment variable like [here](https://github.com/greatnonprofits-nfp/rapidpro/blob/release/3.4/temba/settings_common.py#L1402)?

Comment (teehamaral) [reply temba/settings_common.py:1410]: And I think, for better understanding, would be good to rename this variable to `ALLOW_DUPLICATES_CONTACT_IMPORT` and the default value of it as `True`, and apply the logic on `if urn in seen_urns` statement based on this new name. What do you think?

**Review by teehamaral — CHANGES_REQUESTED** [review]

**Review by teehamaral — APPROVED** [review]

File: temba/contacts/models.py [hunk 2]
```diff
@@ -2148,6 +2151,9 @@ def try_to_parse(cls, org: Org, file, filename: str) -> Tuple[List, int]:
                 seen_uuids.add(uuid)
             for urn in urns:
                 if urn in seen_urns:
```
Comment (teehamaral) [inline temba/contacts/models.py:2153]: I think we can add this statement unique like:

```
if urn in seen_urns:
    if not settings.ALLOW_DUPLICATE_CONTACT_IMPORT:
        raise ValidationError(
            _("Import file contains duplicated contact URN '%(urn)s'."), params={"urn": urn}
        )

    num_duplicates += 1
```
What do you think?

**Review by teehamaral — CHANGES_REQUESTED** [review]

File: temba/contacts/models.py (hunk 2 above)
Comment (musamusa) [reply temba/contacts/models.py:2153]: yup, looks legible that way, got my buried in the tests for a while now

**Review by teehamaral — APPROVED** [review]

**Pull request merged**